modules/moderation.py

```diff
@@ -43,12 +43,12 @@
 
 if TYPE_CHECKING:
     from core.context import Interaction
-    from types_.papi import ModLogPayload, PythonistaAPIWebsocketPayload
+    from types_.papi import ModLogPayload
```

Comment:
Since this isn't technically from the Pythonista API anymore, should we rename all references to it to something else, or are we assuming the Bot webserver *is* the API?

Reply:
Ya we can just assume the webserver is the API.